content/docs/setup/consul/quick-start/index.md

```diff
@@ -48,12 +48,15 @@ For example, run the following command on a macOS or Linux system:
 
 1. Change directory to the root of the Istio installation directory.
 
-1.  Bring up the Istio control plane containers:
+1.  Bring up the Istio networking control plane containers:
 
     {{< text bash >}}
     $ docker-compose -f install/consul/istio.yaml up -d
     {{< /text >}}
 
+    > Note: The Consul install only configures Istio Pilot. To use Istio Mixer (policy enforcement and telemetry reporting) or Istio Galley, further installation steps
```

Comment:
It would be good to update this guide to take into account Consul Connect (which also uses envoy and its control plane). Perhaps, someone from Consul community could help out.

Reply:
Sounds good. I'll file an issue about integrating Mixer into consul-managed envoys.